App/src/components/Request.js

```diff
@@ -2,37 +2,94 @@
 import React from "react";
 import { jsx } from "@emotion/core";
 
-function Request({request, findName, handleRequestSchedule, Turn}) {
+const Turn = {
+  1: "Morning",
+  2: "Afternoon",
+  3: "Night",
+  4: "OFF"
+};
+
+function Request({ request, findName, onChange, active }) {
   return (
     <li
```

Comment:
Don't use a `li`, this should be an `article`
```suggestion
    <article
```

Reply:
Thank you , Done